theme/layout/theme.liquid

```diff
@@ -20,5 +20,22 @@
   </main>
     {% render 'footer' %}
     {{ 'index.js' | asset_url | script_tag }}
+
+    <script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.min.js"></script>
```

Comment:
Kind of sucks that we need to import all of jQuery just to use this slider. I wonder if there is an alternative we can use written in JS. This one looks pretty good: https://splidejs.com/options/

Reply:
@sepowitz that library works well!! (So far)